scripts/src/primazatest/runtest.py

```diff
@@ -1005,10 +1005,12 @@ def main():
                         dest="input_dir",
                         help="directory for kubeconfigs used for user tests",
                         required=False)
-    parser.add_argument("-j", "--sa_namespace",
-                        dest="sa_namespace",
-                        help="directory for kubeconfigs used for user tests",
-                        required=False)
+    parser.add_argument("-j", "--primaza_identity_namespace",
```

Comment:
```suggestion
    parser.add_argument("-j", "--primaza-identity-namespace",
```

Reply:
The _ is there is all the other  options so thought to keep the same way